### ISSUE TITLE
Production name fallback to AGL instead of uniXXXX?

### ISSUE THREAD

**Issue by justvanrossum** [googlefonts/ufo2ft#494]
I keep running into a minor annoyance: as soon as production names are requested, the fallback is uniXXXX names. Is there any particular reason why AGLFNF could not be used instead as a primary fallback?

**Comment by anthrotype** [comment]
Yeah, I'm open to that. Fonttools should have all that already. Can you work on a PR?

**Comment by khaledhosny** [comment]
See also #211
